README.md

```diff
@@ -5,51 +5,51 @@
 [![CircleCI](https://circleci.com/gh/cosmos/voyager.svg?style=svg)](https://circleci.com/gh/cosmos/voyager)
 [![codecov](https://codecov.io/gh/cosmos/voyager/branch/develop/graph/badge.svg)](https://codecov.io/gh/cosmos/voyager)
 
-👋 Welcome to Voyager, the official user interface for the [Cosmos Network](https://cosmos.network/).
-
-💻 Voyager is a desktop application built with [Electron](https://github.com/electron/electron). Voyager runs on macOS 10.9+, Windows 7+, and Debian-based Linux distros.
+👋 Welcome to Voyager, the official desktop application for the [Cosmos Network](https://cosmos.network/).
 
 ⚠️ This is still alpha-level software. **DO NOT** enter your Cosmos fundraiser seed into Voyager.
 
+💻 Voyager runs on macOS 10.9+, Windows 7+, and Debian-based Linux distros.
+
 🎉 Binary releases are [available here](https://github.com/cosmos/voyager/releases).
 
 ---
 
-### Prerequisites
+## Voyager Prerequsities
 
-#### Install dependencies
+### Install Cosmos SDK
```

Comment:
we should use zach's gist.

Reply:
Which one is that? Can you make a new issue with these two comments?